libraries/common/store/index.js

```diff
@@ -40,8 +41,32 @@ export function configureStore(reducers, subscribers) {
     )
   );
 
-  initSubcribers(subscribers);
+  initSubscribers(subscribers);
   syncRouter(store);
 
   return store;
 }
+
+/**
+ * Configures a mocked store for tests.
+ * @param {Function} reducers The reducers to use in the mock store.
+ * @param {Array} subscribers Streams subscribers to initialize.
+ * @returns {Store}
+ */
+export function createMockStore(reducers = null, subscribers = null) {
+  const mockedStore = configureMockStore([thunk, streams]);
+
+  let state;
+
+  if (typeof reducers === 'function') {
+    state = reducers({}, {});
+  }
+
+  const store = mockedStore(state);
+
+  if (subscribers !== null) {
+    initSubscribers(subscribers);
```

Comment:
It would be nice to add `[].concat()` here so that one subscriber doesn't have to be manually passed as an array.